src/config/services/date-time.md

```diff
@@ -0,0 +1,61 @@
+# Date and Time
+
+To maintain accuracy of your system's clock, you can use the [Network Time
+Protocol](https://en.wikipedia.org/wiki/Network_Time_Protocol) (NTP).
+
+Void provides packages for three NTP daemons: NTP, OpenNTPD and Chrony.
+
+Once you have installed an NTP daemon, it can be enabled with:
+
+```
+# ln -s /etc/sv/ntpd /var/service/
```

Comment:
Because people will verbatim copy/paste things without looking, maybe just say that you should enable the service you chose and link to the doc for enabling services. I don't know if we need to repeat the steps for enabling a service here. Documenting the `ntpd` symlink is still important, but should include some mention of it being part of package alternatives, and that users need to verify which alternative is chosen before enabling `ntpd` instead of using the actual service name, such as `chronyd` or `busybox-ntpd`.

Thinking about that, it might be better to tell people the real service name of each daemon below and not have them rely on an ntpd symlink. This is fairly important given that you can also install something like busybox and this will provide another `ntpd` alternative. This could confuse folks who install something and don't expect it to provide an ntpd symlink (confusion is probably the norm for packages using alternatives).

Reply:
I would also like to add that talking about enabling the service (and information on how to) can just be done once, and does not need to be repeated after the description of each different package.

Reply:
@nilium, @bobertlo: Thank you both for your feedback. :-)

Since i didn't mention enabling/disabling the service in the description for each of NTP / OpenNTPD / Chrony, i presume you mean that i should just remove the enabling/disabling section beginning with the sentence "Once you have installed an NTP daemon ...", since enabling and disabling services is now described in runit/managing?

Then, instead of talking about the symlinks from `ntpd` to `<service>`, i should simply note the service name for each of NTP / OpenNTPD / Chrony?

Reply:
re: Enabling / disabling, yes. You can just tell the users to enable the service of their choice and link to the doc on enabling services, and call out the names of each service below. The doc isn't comprehensive (doesn't include busybox, for example), but that's fine as long as the common choices are covered.

Reply:
Thanks - PR updated.